helm-charts/interoperator/values.yaml

```diff
@@ -28,6 +28,23 @@ broker:
   service:
     type: LoadBalancer
 
+quota_app:
+  port: 9296
+  username: quota
+  password: secret
+  quota:
+    enabled: false
+    oauthDomain: https://myauth-domain.com
+    serviceDomain: https://my-tenant-onboarding.com
+    username: quota_user
+    password: quota_password
+  image:
+    repository: servicefabrikjenkins/service-fabrik-broker
+    tag: 0.6.0
+    pullPolicy: Always
+  service:
+    type: LoadBalancer
```

Comment:
Default  service type as `ClusterIP` might be better. Since this is only consumed from within the cluster.